en/docs/remote-apps/connect-remote-app.md

```diff
@@ -0,0 +1,118 @@
+# Connect Remote Applications
+
+If you need to observe an external application that you have written in Ballerina programming language, you can connect it to Choreo.
+
+## Prerequisites
+
+Create an external application in Ballerina programming language. For instructions, see [Ballerina Documentation - Writing Your First Ballerina Program](https://ballerina.io/learn/user-guide/getting-started/writing-your-first-ballerina-program/).
```

Comment:
Why do we say external here. IMO, `Create an application...` is enough as external can lead to different meanings

Reply:
Removed the word "external"